apps/alchemist/test/unit/alchemist/broadway_test.exs

```diff
@@ -32,9 +32,9 @@ defmodule Alchemist.BroadwayTest do
         TDG.create_transformation(%{
           type: "regex_extract",
           parameters: %{
-            "sourceField" => "first_name",
-            "targetField" => "first_letter",
-            "regex" => "^(\\w)"
+            sourceField: "first_name",
+            targetField: "first_letter",
+            regex: "^(\\w)"
```

Comment:
We changed this test in alchemist, then realized the fix probably worked better in transformers. I still think this test is valid and covers it, so we didn't move it 🦐 

Do we need to change the alchemist version for this?

Reply:
Version wouldn't need to be bumped since the tests are not packaged in the binary.

I'm cool with leaving this here as well since the idea is we can handle either case

Reply:
Is there a test case that uses string for the keys as well?

Reply:
Because of the way test_data_generator works, this change can be reverted I think. No matter if the parameters are provided with strings or atoms as keys, the result will have entires with atoms as keys.

``` elixir
iex(1)> string_keys =
...(1)>   SmartCity.TestDataGenerator.create_transformation(%{
...(1)>     type: "regex_extract",
...(1)>     parameters: %{
...(1)>       "sourceField" => "first_name",
...(1)>       "targetField" => "first_letter",
...(1)>       "regex" => "^(\\w)"
...(1)>     }
...(1)>   })
%SmartCity.Ingestion.Transformation{
  id: "b4f3265f-6400-48c6-8de8-d1682eba0c5f",
  name: "Transformation Name",
  parameters: %{
    :regex => "^\\((\\d{3})\\)",
    :sourceField => "Sooty",
    :targetField => "Lucy",
    "regex" => "^(\\w)",
    "sourceField" => "first_name",
    "targetField" => "first_letter"
  },
  sequence: 1..1000,
  type: "regex_extract"
}
iex(2)>
nil
iex(3)> atom_keys =
...(3)>   SmartCity.TestDataGenerator.create_transformation(%{
...(3)>     type: "regex_extract",
...(3)>     parameters: %{
...(3)>       sourceField: "first_name",
...(3)>       targetField: "first_letter",
...(3)>       regex: "^(\\w)"
...(3)>     }
...(3)>   })
%SmartCity.Ingestion.Transformation{
  id: "3fe38690-9e29-40dd-bea1-4f5699e59cf7",
  name: "Transformation Name",
  parameters: %{
    regex: "^(\\w)",
    sourceField: "first_name",
    targetField: "first_letter"
  },
  sequence: 1..1000,
  type: "regex_extract"
}
```